### ISSUE TITLE
Use the loss function configured from the modelwrapper/module

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gate-lf-pytorch-json#37]
Currently this seems to be hard coded in the evaluation method.
